### PR TITLE
Bump rubygem-awesome_print to 1.7.0

### PR DESCRIPTION
**Pull request by akofink**
This is to fix an issue calling awesome_print on a multipart response
string that contains file upload metadata, which causes an IOError due
to awesome_print trying to read data a closed stream. Upgrading the gem
fixes this issue, which was blocking bats tests.

### CODE REVIEW

**Comment by mmoll** [comment]
Is this also needed for Debian/Ubuntu?

File: rubygem-awesome_print/rubygem-awesome_print.spec [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 Summary: Pretty print Ruby objects with proper indentation and colors
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.0.2
-Release: 13%{?dist}
+Version: 1.7.0
+Release: 1%{?dist}
 Group: Development/Languages
 License: MIT
 URL: http://github.com/michaeldv/awesome_print
```
Comment (domcleal) [inline rubygem-awesome_print/rubygem-awesome_print.spec]: This has changed and needs updating.

**Comment by domcleal** [comment]
> Is this also needed for Debian/Ubuntu?

Looks like we only ship a standalone package for 14.04 (version 1.2.0), so we could update it. The gem in the Foreman package should be 1.7.0 already. I can't tell from the description which was affected.

**Comment by mmoll** [comment]
Right, I thought about hammer when I asked. If it's not needed for hammer I'd prefer everything to stay like it is, and rather update the package in upstream Debian first.

**Comment by akofink** [comment]
I noticed the issue in the bats tests, which run on EL6 and EL7. I suspect Debian is also affected. I will try to run bats on Debian to find out.

File: rubygem-awesome_print/rubygem-awesome_print.spec (hunk 1 above)
Comment (akofink) [inline rubygem-awesome_print/rubygem-awesome_print.spec]: @domcleal Please explain what needs updating. (sorry I'm new at packaging)

Comment (mmoll) [inline rubygem-awesome_print/rubygem-awesome_print.spec]: the URL is https://github.com/awesome-print/awesome_print now :mag:

**Comment by akofink** [comment]
@mmoll awesome_print v1.2.0 and v1.6.1 fix the problem on Katello bats as well as the latest version, v1.7.0.

**Comment by mmoll** [comment]
@akofink thanks, then there's no need for action on Debian/Ubuntu :+1:

**Comment by domcleal** [comment]
Merged and building, thanks @akofink.

**Pull request merged**